Makefile

```diff
@@ -68,33 +60,28 @@ PACKAGE_TARGETS = \
 	${PKGDIR}/bin/etcd \
 	${PKGDIR}/bin/flowctl \
 	${PKGDIR}/bin/gazctl \
-	${PKGDIR}/bin/gazette \
-	${PKGDIR}/lib/${LIBROCKS}
+	${PKGDIR}/bin/gazette
 
 ##########################################################################
 # Configure Go build & test behaviors.
 
 # Enable the sqlite3 JSON extension.
 GO_BUILD_TAGS += json1
 
-# Configure for building & linking against our vendored RocksDB library.
-export CGO_CFLAGS      = -I${ROCKSDIR}/include
-export CGO_CPPFLAGS    = -I${ROCKSDIR}/include
-export CGO_LDFLAGS     = -L${ROCKSDIR} -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd
-export LD_LIBRARY_PATH =   ${ROCKSDIR}
-
-# Variable used by librocksdb-sys to discover dynamic RocksDB library.
-# TODO(johnny): Ideally this would be specified as cargo configuration:
-#  https://github.com/rust-lang/cargo/pull/8839
-export ROCKSDB_LIB_DIR = ${ROCKSDIR}
+# Targets which Go targets rely on in order to build.
+GO_BUILD_DEPS = \
+	$(GO_PROTO_TARGETS) \
+	${RUSTBIN}/libbindings.a \
+	${RUSTBIN}/librocks-exp/librocksdb.a \
+	crates/bindings/flow_bindings.h
 
 ##########################################################################
 # Build rules:
 
 # `protoc-gen-gogo` is used to compile Go protobufs.
 ${TOOLBIN}/protoc-gen-gogo:
-	go mod download github.com/golang/protobuf
-	go build -o $@ github.com/gogo/protobuf/protoc-gen-gogo
+	./go.sh mod download github.com/golang/protobuf
+	./go.sh build -o $@ github.com/gogo/protobuf/protoc-gen-gogo
```

Comment:
Not sure if this is a problem, or just something to note:

I had to remove my `.build/tools` directory and rerun `make install-tools` to get `make rust-build` to work. Prior to that, it was failing to find the compiled protobuf stuff. :thinking: